Source/Core/Core/ConfigManager.cpp

```diff
@@ -743,14 +746,15 @@ void SConfig::LoadSettingsFromSysconf()
 
 void SConfig::ResetRunningGameMetadata()
 {
-  SetRunningGameMetadata("00000000", 0, 0);
+  SetRunningGameMetadata("00000000", 0, 0, Core::TitleDatabase::TitleType::Other);
```

Comment:
Yeah, it is. This is the proper commit order, for the reference:

<details>

f8bc392e7c - (HEAD -> show-active-title, fork/show-active-title) UI: Expose the "show active title" setting (52 minutes ago) <Léo Lam>
bdda2c069a - Add an option to show the active title in the title (52 minutes ago) <Léo Lam>
c7df452f5f - WX: Use TitleDatabase for getting title names (52 minutes ago) <Léo Lam>
9aadcd2d60 - Include the latest WiiTDB (52 minutes ago) <Léo Lam>
f535cab404 - Add a TitleDatabase (52 minutes ago) <Léo Lam>

</details>

Each of these commits should even build individually.